docs/deprecated_wiki/Setup/Initial Setup/Security/Security:-Enabling-SSL.md

```diff
@@ -1,20 +0,0 @@
-**[[HOME|Home]]** > **[[SETUP|Setup]]** > **[[SECURITY: ENABLING SSL|Security: Enabling SSL]]**
```

Comment:
it looks like you accidentally deleted this file, which isn't related to the ticket you're trying to fix